code/modules/antagonists/nukeop/equipment/nuclearbomb.dm

```diff
@@ -1,3 +1,5 @@
+var/nuke_code = random_nukecode()
```

Comment:
Global var bad,

Can you do this another way?

Reply:
replace with `GLOBAL_VAR_INIT(nuke_code, random_nukecode())` and add `GLOB.` before all instances of `nuke_code`